packages/core/src/spec/lib/TwaManifestSpec.ts

```diff
@@ -24,6 +24,7 @@ describe('TwaManifest', () => {
         'name': 'PWA Directory',
         'short_name': 'PwaDirectory',
         'start_url': '/?utm_source=homescreen',
+        'display': 'standalone',
```

Comment:
Can we add a tests for a Web Manifest that doesn't have a display mode, using browser, minimal-ui and fullscreen?

Reply:
Can we also add tests for the constructor? We want to check
 - The default value is set when reading a TwaManifestJson that doesn't contain a value for `display`
 - The value is correctly set when reading a TwaManifestJson that contains a `display` value of `fullscreen`